spring-shell2-core/src/main/java/org/springframework/shell2/commands/Help.java

```diff
@@ -107,15 +107,17 @@ private CharSequence documentCommand(String command) {
 			if (description.defaultValue().isPresent()) {
 				result.append("[");
 			}
-			if (!description.mandatoryKey()) {
-				result.append("[");
-			}
-			result.append(description.keys().iterator().next(), AttributedStyle.BOLD);
-			if (!description.mandatoryKey()) {
-				result.append("]");
-				if (!description.formal().isEmpty()) {
-					result.append(" ");
+			if(!description.keys().isEmpty()) {
+				if (!description.mandatoryKey()) {
+					result.append("[");
 				}
+				result.append(description.keys().iterator().next(), AttributedStyle.BOLD);
+				if (!description.mandatoryKey()) {
+					result.append("]");
+				}
+			}
+			if (!description.formal().isEmpty()) {
```

Comment:
Let me check. Without this it was doing `keyformal` without the space: `key forrmal`